stack_printing.py

```diff
@@ -27,6 +27,11 @@ def print_all_stacks(signum, frame):
     for task in asyncio.all_tasks():
         task.print_stack()
```

Comment:
FWIW, I double checked this goes to stdout and not stderr:
![image](https://github.com/viamrobotics/board_canaries/assets/71458/aa1f0d0d-6d7b-4e1f-b5ce-277f4bb1b168)